test/functional/terminal/tui_spec.lua

```diff
@@ -151,7 +151,7 @@ describe('tui', function()
   it('can handle arbitrarily long bursts of input', function()
     execute('set ruler')
     local t = {}
-    for i = 1, 3000 do
+    for i = 2000, 3000 do
```

Comment:
Yeah of course, I just wanted to see if that's the issue here... doesn't change anything though!